index.js

```diff
@@ -3,44 +3,15 @@ const jwt = require('jsonwebtoken');
 const request = require('request');
 const _ = require('lodash');
 const getPem = require('rsa-pem-from-mod-exp');
-let publicKeys = {};
-
-// verify the jwtToken against the given configuration
-exports.verify = function(jwtToken, config) {
-
-    return new BbPromise(function(resolve, reject) {
-        let decoded = jwt.decode(jwtToken, { complete: true });
-        if (!decoded) {
-            reject('{ "status":"error", "message":"Error Decoding JWT Token" }');
-        } else {
-            let jwtKid = decoded.header.kid;
-            if (!jwtKid) {
-                reject('{ "status":"error", "message":"Invalid JWT Token" }');
-            } else {
-                getPublicKeys(config.JWK_URI, jwtKid).then(function(response) {
-                    if (hasPublicKey(jwtKid)) {
-                        let publicKey = getPublicKey(jwtKid);
-                        verifyJwt(jwtToken, publicKey, config.AUD, config.ISS).then(function(response) {
-                            resolve(JSON.stringify({ "status": "success", "message": response }));
-                        }).catch(function(error) {
-                            reject(JSON.stringify({ "status": "error", "message": error }));
-                        });
-                    } else {
-                        reject('{ "status":"error", "message":"Invalid jwt kid" }');
-                    }
-                }).catch(function(error) {
-                    reject('{ "status":"error", "message":"Cannot fetch data from JWK_URI" }');
-                })
-            }
-        }
 
-    });
-}
+let publicKeys = {};
 
 // Validate the jwt Token with the audience and the issuer
 let verifyJwt = function(jwtToken, publicKey, aud, iss) {
```

Comment:
Consider using named function expressions instead of function declarations.
For instance `const verifyJwt = function verifyJwt(...) { ... }`